automated/linux/docker-prune/prune.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# shellcheck disable=SC1091
+. ../../lib/sh-test-lib
+OUTPUT="$(pwd)/output"
+RESULT_FILE="${OUTPUT}/result.txt"
+export RESULT_FILE
+APP="shellhttpd"
+
+check_prune () {
+    runtime="5 minute"
+    endtime=$(date -ud "$runtime" +%s)
+    FOUND=0
+
+    while [ ! "${FOUND}" -eq 1 ]; do
+        if [ "$(date -u +%s)" -ge "$endtime" ]; then
+            break
+            return 1
+        elif (journalctl --no-pager -u aktualizr-lite | grep "Pruning unused docker containers"); then
+            echo "Prune Successful"
```

Comment:
I'd like a more detailed check here. Please see which containers/layers are present before and after the prune.